lightning/checkpoints/checkpoints.go

```diff
@@ -364,12 +365,12 @@ func (*NullCheckpointsDB) InsertEngineCheckpoints(_ context.Context, _ string, _
 func (*NullCheckpointsDB) Update(map[string]*TableCheckpointDiff) {}
 
 type MySQLCheckpointsDB struct {
-	db      *sql.DB
-	schema  string
-	session uint64
```

Comment:
any reason to remove session？ don't we use it forever?

Reply:
For `node_id` it's explained in the PR description. 

For `session` it's just renamed to `task_id`.

Reply:
In fact I don't understand the description, why must need `create select`, `node id` can be just a label

Reply:
In the original design, multiple nodes are sharing the same table distinguished by node_id.

<table><tr><td>checkpoints.table_v5

| node_id | ... |
|---------|-----|
| 1       | A   |
| 1       | B   |
| 2       | C   |
| 2       | D   |

</td></tr></table>

To support keep-after-success + server mode, we'd want to extract the content like this:

<table><tr><td>checkpoints.table_v5

| node_id | ... |
|---------|-----|
| 2       | C   |
| 2       | D   |

</td><td>checkpoints_backup.table_v5

| node_id | ... |
|---------|-----|
| 1       | A   |
| 1       | B   |

</td></tr></table>

So that future checkpoints from node 1 won't interfere with the previous checkpoint, meanwhile the checkpoints of node 2 are not affected. 

The CREATE SELECT is needed to copy content to `checkpoints_backup.table_v5`.

Reply:
If you have a distributed job mode in the future, do you still have to modify it? This modification is confusing, there may be many feasible solutions, but it's not key problem